### PR TITLE
chore: Use backported versions for class copiers fix

### PR DESCRIPTION
**Pull request by hectoras**
**Associated Jira issue:**  [ADF-1148](https://oat-sa.atlassian.net/browse/ADF-1148)

Verified the target branch is in-sync with the latest August release when this PR was opened:
- https://github.com/oat-sa/tao-community/compare/2022.08.2...release-2022-08-lts
- https://github.com/oat-sa/tao-community/compare/release-2022-08-lts...2022.08.2

**TODO**
- [x] Update MediaManager to use a backport based on v12.29.0 instead
  - https://github.com/oat-sa/extension-tao-mediamanager/pull/418
- [x] Repeat the taoUpdate check

<details>
  <summary>Result for taoUpdate</summary>

```
root@06692d00a2ed:/var/www/html# php tao/scripts/taoUpdate.php 
Running extension update
  generis already up to date
  tao requires update from 50.18.3.0 to 50.18.3.1
    Successfully updated tao to 50.18.3.1
  taoResultServer already up to date
  taoOutcomeRds already up to date
  taoDelivery already up to date
  taoBackOffice already up to date
  taoTestTaker already up to date
  taoGroups already up to date
  taoItems requires update from 11.30.0.0 to 11.30.1.0
    Successfully updated taoItems to 11.30.1.0
  taoTests already up to date
  taoQtiItem already up to date
  taoQtiTest already up to date
  taoDeliveryRdf already up to date
  taoOutcomeUi already up to date
  taoQtiTestPreviewer already up to date
  qtiItemPci already up to date
  funcAcl already up to date
  taoCe already up to date
  taoLti already up to date
  taoLtiBasicOutcome already up to date
  taoTaskQueue already up to date
  taoAltResultStorage already up to date
  taoEventLog already up to date
  taoMediaManager requires update from 12.29.0.0 to 12.29.0.1
    Successfully updated taoMediaManager to 12.29.0.1
  taoProctoring already up to date
  taoRevision already up to date
  taoClientDiagnostic already up to date
  pciSamples already up to date
  ltiDeliveryProvider already up to date
  qtiItemPic already up to date
  taoDacSimple requires update from 7.7.0.0 to 7.7.2.0
    Successfully updated taoDacSimple to 7.7.2.0
  
 [OK] Already at the latest version ("oat\taoDacSimple\migrations\Version202107121456123210_taoDacSimple")

  Successfully updated 49 client translation bundles
  Post update actions:
    No actions to be executed
  Update ID : 62fe1353b4d1d
  Update completed
  Dependency Injection Container rebuilt
  FeatureFlag cache cleared
root@06692d00a2ed:/var/www/html# 
```
</details>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
         "oat-sa/extension-tao-lti": "15.8.2",
         "oat-sa/extension-tao-ltideliveryprovider": "12.12.1",
         "oat-sa/extension-tao-revision": "10.4.0",
-        "oat-sa/extension-tao-mediamanager": "12.29.0",
+        "oat-sa/extension-tao-mediamanager": "12.29.2",
```
Comment (gabrielfs7) [inline composer.json:57]: In the version 12.29.1 we generated new bundles with different tao-core version, so this update will cause JS issues

**Review by jsconan — APPROVED** [review]
LGTM!

**Review Checklist**
- [ ] ~~New code is covered by tests (if applicable)~~
- [ ] ~~Tests are running successfully (old and new ones) on my local machine (if applicable)~~
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

File: composer.json (hunk 1 above)
Comment (jsconan) [reply composer.json:57]: It should not be a problem. The code did not change, so the bundles are equivalent. The only difference is that meanwhile, the drop of IE support was effective in the bundler toolchain. Comparing both the commits, the only differences are in the way the code was transpiled.

Comment (gabrielfs7) [reply composer.json:57]: But `12.29.1` was generated using some tao-core from September release and the code in tao-community is from august release. I am not sure this is safe.

Comment (hectoras) [reply composer.json:57]: https://github.com/oat-sa/extension-tao-mediamanager/pull/418

Comment (hectoras) [reply composer.json]: Updated composer to use the new MediaManager backport: 6878b5fee418e067ec02d726d5cc7ee943c62878

I'll test the taoUpdate procedure next

**Comment by hectoras** [comment]
Released as [2022.08.3](https://github.com/oat-sa/tao-community/releases/tag/2022.08.3).

**Pull request merged**